service/src/main/java/uk/gov/hmcts/reform/fpl/enums/HearingNeedsBooked.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum HearingNeedsBooked {
```

Comment:
minor: maybe `hearingBookingNeeds`?

Reply:
ccd fixed list is already called HearingNeedsBooked so not going to change